content/rs/administering/active-active.md

```diff
@@ -39,15 +39,11 @@ Before configuring a CRDB, you must:
 ## CRDB Current Limitations
 
 1. RS is limited to five Participating Clusters or CRDB Instances per CRDB.
-1. An existing database cannot modified to be a CRDB.
-    To move existing data to a CRDB you must create a new CRDB and migrate your data.
+1. To move data from an existing database to a CRDB you must create a new CRDB and migrate the data.
 1. CRDBs do not support [Redis modules]({{< relref "/rs/developing/modules/_index.md" >}}).
 1. CRDBs require FQDNs or mDNS (development only). Discovery Service is not supported with CRDBs.
 1. CRDBs are not compatible with [Replica Of]({{< relref "/rs/administering/active-passive.md" >}}).
 
-    ReplicaOf is a one-way replication, while CRDB utilize multi-master replication.
```

Comment:
Leave row 48... is it part of the previous line.

Reply:
I took it out because replica of is described in the replica of article. I don't think we need to explain about it here.